developer_docs/installation.md

```diff
@@ -29,6 +29,9 @@ Follow these instructions to set up your development environment, which you need
 11. Install the [React Developer Tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en)
 12. Open and close the Redux DevTools using `ctrl+h`, and move them with `ctrl+w`
 
+## Tests (in progress)
```

Comment:
Small documentation thing—could this be in `developer_docs/development`? I think that makes more sense. If you can, it would also be awesome if this were fleshed out a little bit more, even just a sentence that this uses Jest. Maybe a good starting point is the [p5.js documentation about unit tests](https://github.com/processing/p5.js/tree/master/developer_docs#unit-tests).

Reply:
I added some basic documentation. I am going to do another pass and add some more test and update the docs accordingly. Do you have any suggestion about any particular part of the code that would benefit the most from the unit tests?

Reply:
👍 That might need more than unit tests to verify but will look into it.